getting-started.md

```diff
@@ -892,30 +894,38 @@ spec:
       - name: first-key
 ```
 
-
-When a developer deploys an application with a matched image name and the image is signed:
-
+When a developer deploys a runnable resource with an image name that matches a
```

Comment:
Suggestion: When a developer deploys a runnable resource whose container image matches a pattern in the ClusterImagePolicy, and that image is signed with the correct signature:

Reply:
Would it be better to say:
> and that image is signed with a matching key
instead of
> and that image is signed with the correct signature

I'm wondering because "correct signature" may imply there's a way of generating a "wrong signature", but we may want this wording to be more around keys that match or don't match in this context.

Reply:
How about "and that image is signed with an expected signature"?

I was thinking that using "expected" implies that the pubkey was configured in the ClusterImagePolicy

Reply:
Makes sense, I think "expected" delivers the meaning I was hoping for. 😃